README.md

```diff
@@ -2,21 +2,37 @@
 
 # Deconst documentation
 
-Documentation for the *deconst* project: a continuous delivery pipeline for heterogenous documentation. It's also used to develop deconst itself, like some kind of documentation ouroboros. You can read the documentation itself at [deconst.horse](https://deconst.horse/).
+Documentation for the *deconst* project: a continuous delivery
+pipeline for heterogenous documentation. It's also used to develop
+deconst itself, like some kind of documentation ouroboros. You can
+read the documentation itself at
+[deconst.horse](https://deconst.horse/).
 
 ## Building
 
-To build this documentation standalone, use the [Deconst client](https://github.com/deconst/client). Use the Sphinx preparer and a clone of `https://github.com/deconst/deconst-docs-control` as the control repository.
+To build this documentation standalone, use the [Deconst
```

Comment:
We should actually remove info around the Client since it's broken. Want to do that here?

Reply:
Preference is to leave this PR just about line length.

AFA the Client, do we ever plan to fix it? Here's an issue to track the discussion: https://github.com/deconst/deconst-docs/issues/261